--- venv/lib/python3.12/site-packages/aiohttp/web_urldispatcher.py	2024-08-01 13:01:32
+++ /tmp/web_urldispatcher.py	2024-08-01 22:37:19
@@ -1132,15 +1132,7 @@
     def _get_resource_index_key(self, resource: AbstractResource) -> str:
         """Return a key to index the resource in the resource index."""
         # strip at the first { to allow for variables
-        #return resource.canonical.partition("{")[0].rstrip("/") or "/"
-        if "{" in (index_key := resource.canonical):
-            # strip at the first { to allow for variables, and than
-            # rpartition at / to allow for variable parts in the path
-            # For example if the canonical path is `/core/locations{tail:.*}`
-            # the index key will be `/core` since index is based on the
-            # url parts split by `/`
-            index_key = index_key.partition("{")[0].rpartition("/")[0]
-        return index_key.rstrip("/") or "/"
+        return resource.canonical.partition("{")[0].rstrip("/") or "/"
 
     def index_resource(self, resource: AbstractResource) -> None:
         """Add a resource to the resource index."""
